orca-web/src/test/groovy/com/netflix/spinnaker/orca/controllers/TaskControllerSpec.groovy

```diff
@@ -248,7 +248,7 @@ class TaskControllerSpec extends Specification {
         pipelineConfigId = config.pipelineConfigId
       }
     })
-    front50Service.getPipelines(app, false) >> [[id: "1"], [id: "2"]]
+    front50Service.getPipelines(app, false, null) >> [[id: "1"], [id: "2"]]
```

Comment:
I see null tests, but be nice to verify that it's reading the values & passing them downstream as planned.  Not "critical" though.

Reply:
Sure, pushed an updated test that shows every scenario